### ISSUE TITLE
moduleName generates a not valid query selector

### ISSUE THREAD

**Issue by alexberazouski** [vaadin/vaadin-themable-mixin#40]
In some cases `moduleName` in https://github.com/vaadin/vaadin-themable-mixin/blob/20dc7751f225cda9631fccceb90e12abd4d19f74/vaadin-themable-mixin.html#L64 contains a space and that breaks the selector query (adding quotes around the `moduleName` fixes an issue).

Is this value supposed to be a single word always and my project is wrongly configured?

In our case the `moduleName` value is `lumo-required-field lumo-field-button`

The way to reproduce is to create an empty polymer project and add `vaadin-login` bower dependency.

**Comment by web-padawan** [comment]
Thanks, it's definitely a bug and should be fixed with a test covering it.

**Comment by tomivirkki** [comment]
Tried to reproduce by creating an empty polymer project and adding vaadin-login bower dependency. Are there some additional changed I need to do?

https://glitch.com/edit/#!/glitter-metal

**Comment by alexberazouski** [comment]
Actually, a smaller example will be:
```
"vaadin-text-field": "vaadin/vaadin-text-field#^2.3.0-alpha7"
```

And then add to your initiated component:
```
<link rel="import" href="../../vaadin-text-field/src/vaadin-text-field.html">
<link rel="import" href="../../vaadin-text-field/src/vaadin-password-field.html">
...
<vaadin-text-field></vaadin-text-field>
<vaadin-password-field></vaadin-password-field>
```

By the way, if password-field is placed first, exception is not thrown

**Comment by tomivirkki** [comment]
Can you provide a live example where this occurs? I tried the above at https://jsbin.com/jutumezimo/edit?html,output and still no problems

**Comment by Artur-** [comment]
```
bower init
bower install vaadin-login#1.0.0-alpha7
```
index.html:
```
<!doctype html>
<head>
<link rel="import" href="../vaadin-text-field/vaadin-text-field.html">
<link rel="import" href="../vaadin-text-field/vaadin-password-field.html">
<body>
<vaadin-text-field></vaadin-text-field>
<vaadin-password-field></vaadin-password-field>
```

```
polyserve --open
```

**Comment by manolo** [comment]
@web-padawan, please, can you do a new release of the mixin with the fix?

**Comment by web-padawan** [comment]
Sure, will do and backport to 1.3

**Comment by web-padawan** [comment]
Released 1.4.1 and 1.3.3 with the fix.
